telemetry/zap_log.go

```diff
@@ -0,0 +1,43 @@
+package telemetry
+
+import (
+	"context"
+
+	"github.com/uptrace/opentelemetry-go-extra/otelzap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// NewZapLog returns a Log that carries a zap logger..
+func NewZapLog(logger *zap.Logger) Log {
+	return &otelZapLog{
```

Comment:
Lets set the debug level so that we can configure via env

Reply:
That's not the level of the logger, that's the minimum level that  `otelzap` considers a log to be part of a span: https://github.com/uptrace/opentelemetry-go-extra/blob/main/otelzap/otelzap.go#L440

The log level gets set outside of anything in `zazen` since the logger is passed to us with the level already set so apps are free to config that however they want.